src/app/api/run/route.test.ts

```diff
@@ -0,0 +1,65 @@
+import { describe, it, expect, beforeEach, afterEach } from 'vitest'
+import fs from 'fs'
+import path from 'path'
+import { UPLOAD_DIR } from '@/app/utils'
+import { GET } from '@/app/api/run/route'
+
+describe('GET /api/run', () => {
+    const testFiles = ['1.csv', '2.csv', '3.csv', 'empty.csv']
```

Comment:
I just happened to notice this, but this test seems to be failing. A couple of related comments:

* I believe that's because it doesn't reflect that the app writes the files without file extension and just the `runId`
* Right now the CI is failing at linting and doesn't get to the tests. For SA we did [this](https://github.com/safeinsights/setup-app/pull/2/files) after hitting something similar so we don't get in the habit of otherwise ignoring the :x: :joy: